lottie/src/main/java/com/airbnb/lottie/LottieTask.java

```diff
@@ -154,7 +144,7 @@ private synchronized void notifySuccessListeners(T value) {
     }
   }
 
-  private void notifyFailureListeners(Throwable e) {
+  private synchronized void notifyFailureListeners(Throwable e) {
```

Comment:
The call to `notifySuccessListeners` is synchronized, so it seems like this should be too. Not sure if it was just an oversight?